python/src/aac/lang/active_context_lifecycle_manager.py

```diff
@@ -21,11 +30,29 @@ def get_active_context(reload_context: bool = False) -> LanguageContext:
     global ACTIVE_CONTEXT
 
     if not ACTIVE_CONTEXT or reload_context:
-        ACTIVE_CONTEXT = get_initialized_language_context()
+        ACTIVE_CONTEXT = load_language_context(__state_file_name__, reload_context)
 
     return ACTIVE_CONTEXT
 
 
+def load_language_context(file_name: str, reload_context: bool = False) -> LanguageContext:
+    """
+    Load the language context from filename.
+
+    Args:
+        file_name (str): The name of the file from which to load the language context.
+        reload_context (bool): If True, a fresh active context will be returned; otherwise, the
+            active context will be loaded from the provided file URI if it exists.
+
+    Returns:
+        A language context object loaded from the file, if it exists; otherwise, None.
+    """
+    if lexists(file_name) and not reload_context:
+        return get_language_context_from_state_file(file_name)
+    else:
+        return get_initialized_language_context()
```

Comment:
I had this originally but I changed it because we still want to return a valid language context if the state file doesn't exist so either way `load_language_context` would return `get_initialized_language_context()` if the file doesn't exist.

To me, the approach I settled on seemed a bit cleaner to account for both cases.

Reply:
After discussing it, I'll work on moving the `import`/`export` functionality to the `LanguageContext` class. Doing it this way, would circumvent the entire issue.